.eslintrc.js

```diff
@@ -15,6 +21,10 @@ module.exports = {
     browser: true
   },
   rules: {
+    'ember/no-jquery': 'error',
+    // disable the following until Octane conversion
+    'ember/no-observers': 'off',
+    'ember/no-new-mixins': 'off'
```

Comment:
I have a separate issue for updating this package to Octane conventions. Disabling these for now to prevent spurious test failures.